### ISSUE TITLE
Jetpack Activity content can be truncated in landscape orientation with large fonts

### ISSUE THREAD

**Issue by twstokes** [wordpress-mobile/WordPress-iOS#19128]
### Expected behavior
View is scrollable, enabling all content to be seen.

### Actual behavior
View is not scrollable, truncating content.

### Steps to reproduce the behavior

Prerequisites:
- A Jetpack connected site
- At least one item in the Jetpack Activity Log (you can change the site title to generate one)
- System set to large accessibility fonts

Steps to test:
1. On My Site, tap "Activity Log"
2. Tap an item in the Activity Log
3. Rotate to landscape
4. Observe that the content is truncated and the view cannot be scrolled

| Portrait | Landscape |
| - | - |
| ![Simulator Screen Shot - iPhone 13 - 2022-07-29 at 14 07 31](https://user-images.githubusercontent.com/2092798/181819052-17e2c902-aaa9-42f7-b8c4-4b07dd802cea.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-29 at 14 07 36](https://user-images.githubusercontent.com/2092798/181819068-9a290cb8-768a-46c3-a3ba-d565fdba9ce9.png) |

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
